### ISSUE TITLE
Recommended latest image is too large for most clusters

### ISSUE THREAD

**Issue by erikkrieg** [go-skynet/helm-charts#30]
While trying out the helm chart I found that the recommended image (from the example values file) is far too large to pull on our cluster. Pulling the image locally I'm seeing that it is just shy of 70.0 Gi.

I think that if the docs make some suggestions for different image tags it would make installing the chart easier.

I'm going forward with `v2.0.0-ffmpeg-core`.

**Comment by ronaldpetty** [comment]
Agree. It would also just be good to have a small default (does minimum) just to make sure everything can be installed.

**Comment by ssurovich** [comment]
Great point - On a standard K8s node, this is going to really fill up the overlay and when that node hits >=80% you will start getting evictions.  Most orgs allocate 100-200GB on the overlay (from what I have seen) - so this taking up 1/4 of the storage may cause issues to other workloads.   One thing you could do is to add a node with a taint and then use a toleration to limit that node to running just localai (and other resources for it).   I know that can be easier said than done...

**Comment by mudler** [comment]
docs about the different image tags is here: https://localai.io/basics/getting_started/#container-images - I do agree generally, however note that after few rounds of enhancements the images now are much smaller, compressed are in order of ~17GB:

![Screenshot from 2024-02-26 14-39-12](https://github.com/go-skynet/helm-charts/assets/2420543/31b54660-80b4-4cc1-ae85-828474d3b27e)

**Comment by ssurovich** [comment]
Nice progresss on optimizing - I do get why they are larger images in general - and using a PVC for the models is really the way to help any stress on the conainer host filesystem.
